doc/code/qml_qnn.rst

```diff
@@ -3,7 +3,16 @@ qml.qnn
 
 .. currentmodule:: pennylane.qnn
 
-.. automodapi:: pennylane.qnn
-    :no-heading:
-    :no-inheritance-diagram:
-    :no-inherited-members:
+This module contains functionality for converting PennyLane QNodes into layers that are compatible
+with Keras and PyTorch.
+
+.. note::
+
+    Check out our `Keras <https://pennylane.ai/qml/demos/tutorial_qnn_module_tf.html>`__
+    and `Torch <https://pennylane.ai/qml/demos/tutorial_qnn_module_torch.html>`__ tutorials for
```

Comment:
Is this the correct way to link?

Reply:
I think you can use

```rest
:doc:`/demos/tutorial_qnn_module_torch`
```
as intersphinx is set up :) You may need to remove the leading slash before `demos/` not entirely sure.